main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -490,15 +490,15 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
     }
 
     def Equality: Rule1[ParsedAst.Expression] = rule {
-      Relational ~ optional(optWS ~ (operator("==") | operator("!=") | operator("<=>")) ~ optWS ~ Relational ~ SP ~> ParsedAst.Expression.Binary)
+      Relational ~ zeroOrMore(optWS ~ (operator("==") | operator("!=") | operator("<=>")) ~ optWS ~ Relational ~ SP ~> ParsedAst.Expression.Binary)
```

Comment:
Why? You want to allow `x == y == z`? It does not mean what you think! :)

Reply:
Somewhere in the standard library we have something like `x <=> y == GreaterThan`, which prompted me to make this change. But I agree this is a bad idea! I'll revert this and refactor that Stdlib function. (And add a comment about why it's not zeroOrMore)